docs/acknowledgements.txt

```diff
@@ -88,6 +88,10 @@ provide you with official support from Eaton, or a better level of device
 support in NUT.*
 ================================================================================
 
+* link:https://www.ametek.com/products/businessunits/powersystemsandinstruments/powervar[AMETEK
+Powervar], through Andrew McCartney, has added support for all AMETEK Powervar
+UPM models as usb-hid UPS.
+
```

Comment:
Note @andrewmccartney : I took the liberty to stub an acknowledgements entry here. Feel free to modify if I got some names or spellings wrong.